languages/rust/exercises/concept/tuples/.meta/example.rs

```diff
@@ -0,0 +1,66 @@
+#[derive(Debug, PartialEq, Clone, Copy)]
+pub enum Unit {
+    Millilitre,
+    UsCup,
+    UsFluidOnce,
+    UsTeaspoon,
+    UsTablespoon,
+}
+
+pub fn get_volume(volume: &(Unit, f32)) -> f32 {
+    volume.1
+}
```

Comment:
I want to expand on this a little. The whole point of tuples is for cases where it's too tedious to bother writing a named struct. However, if you ever find yourself in a situation where it's impractical to use pattern-matching and error-prone to use field-access notation, the proper solution isn't to write a `get_field` helper; it's to refactor your tuple into a struct with named fields. 

In a real application for this scenario, it would be better form to define a `struct Measurement {qty: f32, unit: Unit}` anyway. Given the limitations of teaching as fundamental a concept as tuples, there's some leeway to teach unrealistically. Still, this function goes too far: a helper function of this nature should never pass code review, and therefore, we shouldn't be teaching it.